nff/io/ase_calcs.py

```diff
@@ -85,6 +86,18 @@ def __init__(
         self.model_units = model_units
         self.prediction_units = prediction_units
 
+        print("Requested properties:", self.properties)
+        
+        # Unit conversion factors
+        self.energy_units_to_eV = 1.0  # assuming default model outputs energy in eV
+        self.length_units_to_A = 1.0   # assuming default model uses A**3 for length
+
+        # Adjust conversion factors if the model uses different units
+        if self.model_units == "kcal/mol":
+            self.energy_units_to_eV = 0.0433641  # kcal/mol -> eV
+        if self.prediction_units == "nm":
+            self.length_units_to_A = 10.0  # nm -> AA*3
```

Comment:
It might be best to define these unit conversions near the beginning of the file, similar to the constant `HARTREE_TO_EV`, although some of these conversions might already be defined in the ASE `units` module or the NFF `constants` module, which could then be imported and used for clarity.